Formula/ceylon.rb

```diff
@@ -3,19 +3,20 @@ class Ceylon < Formula
   homepage "https://ceylon-lang.org/"
   url "https://ceylon-lang.org/download/dist/1_3_3"
   sha256 "4ec1f1781043ee369c3e225576787ce5518685f2206eafa7d2fd5cfe6ac9923d"
+  revision 1
 
   bottle :unneeded
 
-  depends_on :java => "1.7+"
+  depends_on :java => "1.8"
 
   def install
     man1.install Dir["doc/man/man1/*"]
     doc.install Dir["doc/*"]
     libexec.install Dir["*"]
-
-    # Symlink shell scripts but not *.plugin
-    bin.install_symlink "#{libexec}/bin/ceylon"
-    bin.install_symlink "#{libexec}/bin/ceylon-sh-setup"
+    bin.mkdir
+    mv "#{libexec}/bin/ceylon", bin
```

Comment:
If you use `bin.install` instead of `mv` it should handle the `mkdir` part for you?

Reply:
Amended.

Reply:
Thanks! ❤️